docs/sources/articles/configuring.md

```diff
@@ -1,17 +1,36 @@
-page_title: Configuring Docker
-page_description: Configuring the Docker daemon on various distributions
-page_keywords: docker, daemon, configuration
+page_title: Configuring and running Docker
+page_description: Configuring and running the Docker daemon on various distributions
+page_keywords: docker, daemon, configuration, running, process managers
 
-# Configuring Docker on various distributions
+# Configuring and running Docker on various distributions
 
-After successfully installing Docker, the `docker` daemon runs with it's default
-configuration. You can configure the `docker` daemon by passing configuration
-flags to it directly when you start it.  
+After successfully installing Docker, the `docker` daemon runs with its default
+configuration.
 
 In a production environment, system administrators typically configure the
 `docker` daemon to start and stop according to an organization's requirements.  In most
 cases, the system administrator configures a process manager such as `SysVinit`, `Upstart`,
-or `systemd` to manage the `docker` daemon's start and stop.
+, `systemd` to manage the `docker` daemon's start and stop.
+
+### Running the docker daemon directly
+
+The `docker` daemon can be run directly using the `-d` option. By default it listens on
+the Unix socket `unix:///var/run/docker.sock`
+
+    $ docker -d
+
+    INFO[0000] +job init_networkdriver()
+    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
+    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
+    ...
+    ...
+
+### Configuring the docker daemon directly
+
+If you're running the `docker` daemon directly by running `docker -d` instead
+of using a process manager, you can append the configuration options to the `docker` run
```

Comment:
James Turnbull suggested using this in an earlier note.
ᐧ

On Wed, May 6, 2015 at 6:03 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In docs/sources/articles/configuring.md
> <https://github.com/docker/docker/pull/12804#discussion_r29733082>:
>
> > +
> > +The `docker` daemon can be run directly using the `-d` option. By default it listens on
> > +the Unix socket `unix:///var/run/docker.sock`
> > +
> > +    $ docker -d
> > +
> > +    INFO[0000] +job init_networkdriver()
> > +    INFO[0000] +job serveapi(unix:///var/run/docker.sock)
> > +    INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
> > +    ...
> > +    ...
> > +
> > +### Configuring the docker daemon directly
> > +
> > +If you're running the `docker` daemon directly by running `docker -d` instead
> > +of using a process manager, you can append the configuration options to the `docker` run
>
> I think "run" should be removed here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12804/files#r29733082>.
>



-- 
Ankush Agarwal
http://ankush.io
@ankushio <https://twitter.com/ankushio>

*There are 10 types of people in this world. Those who understand binary
and those who don't.*